netci.groovy

```diff
@@ -781,6 +866,14 @@ combinedScenarios.each { scenario ->
                                         buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} skiptestbuild"
                                         buildCommands += "tests\\buildtest.cmd ${lowerConfiguration} ${architecture} ilasmroundtrip"
                                     }
+                                    else if (scenario == 'r2r') {
+                                        buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} docrossgen skiptestbuild"
```

Comment:
@gkhanna79 looking at build.cmd it looks like both are accepted, but "skiptestbuild" is the old version which is accepted for backwards compatibility. I'll make the change.